includes/class-wc-beta-tester.php

```diff
@@ -391,4 +391,89 @@ public function get_version_information( $version ) {
 
 		return $github_data;
 	}
+
+	/**
+	 * Return true if version string is a beta version.
+	 *
+	 * @param string $version_str Version string.
+	 * @return bool
+	 */
+	protected static function is_beta_version( $version_str ) {
```

Comment:
Do we need these to be static given the class is singleton?

Reply:
We don't need it, I just have a habit to mark functions that don't depend on/alter object's state as static. Is there a drawback to have this as static in php?

Reply:
No drawbacks but I believe it's generally considered a code smell (mixture of static/non-static methods). I could be wrong though :)